monero-outreach-docs/translations/es/archive/wallet_primer_es.md

```diff
@@ -0,0 +1,103 @@
+
+_Clave privada de visualización:_
+**c072ce1678325c1fc09f06ed3f72bdc 290b675747cc01d5255773a63604e8a01**
+
+Al igual que la clave privada de gasto, el penúltimmo dígito hexadecimal de la clave privada de visualización siempre es 0 y muy rara vez 1. 
+
+La clave pública (también llamada dirección) es más larga y usualmente se muestra como un hilo (string) en vez de un hexadecimal. Aquí está la clave pública correspondiente a la clave privada de gasto dada anteriormente:
+
+_Clave pública (dirección):_
+**44XiEedj1ivSCY6A1dFCDCCxZ3nxUQyD aGauuvgYNCAjbKSPzCe3RQ3goiXVvmfX axdYecaBQVNHPGszK9Jmp7Lg3SPAMr1**
+
+(¡No envíen Monero a esta dirección! Cualquiera que conozca la clave privada de gasto, en otras palabras, cualquiera que lea este artículo, puede cogerlos.)
+
+La clave pública es un hilo (string) de 95 caracteres que empieza con el número 4. Está compuesto por 58 tipos de caracteres que son seleccionados para ser visualmente claros (no tendrás inconvenientes, por ejemplo, con el dígito 0 o la mayúscula O). Tiene restricciones internas que ordenan que aquellos caracteres aleatorios o errores usualmente no formen una dirección válida.
+
+El proceso por el cual estas tres claves están hechas con software se muestra en el siguiente diagrama:
+
+			   Frase Mnemónica
+			        ↑↓
+ Datos Aleatorios ----> Clave Privada de Gasto ----> Clave Privada de Visualización
+                                      |
+                                      |
+				          |    ----> Clave Pública (Dirección)
+
+El proceso criptográfico hace que las flechas rojas en el diagrama anterior sea irreversible. El cálculo de la clave privada de gasto no es posible conociendo la clave pública y la clave privada de visualización.
+
+### _Generando claves de Monero_
+
+Puedes crear claves basado en el proceso anterior utilizando 1. software de monederos, 2. sitios web de propósito especial (special-purpose websites) o 3. monederos de hardware.
+
+**Generación de claves método 1:** Casi todos los monederos de software pueden ser usados para generar claves, pero debes debes confiar en que el software sea competente y honesto. Es más seguro usar el monedero oficial de Monero o un monedero recomendado por el sitio web de Monero, [web.getmonero.org](https://web.getmonero.org/es/). Monero Outreach tiene las mismas recomendaciones iguales a las del sitio web oficial en la [Guía de Inicio Rápido para el Monedero de Monero](https://www.monerooutreach.org/cuentos/monedero-de-monero.html).
```

Comment:
```suggestion
**Generación de claves método 1:** Casi todos los monederos de software pueden ser usados para generar claves, pero debes debes confiar en que el software sea competente y honesto. Es más seguro usar el monedero oficial de Monero o un monedero recomendado por el sitio web de Monero, [web.getmonero.org](https://www.getmonero.org/es/). Monero Outreach tiene las mismas recomendaciones iguales a las del sitio web oficial en la [Guía de Inicio Rápido para el Monedero de Monero](https://www.monerooutreach.org/cuentos/monedero-de-monero.html).
```

It's better to use `www.getmonero` instead of `web.getmonero`. The canonical address was recently changed from `https://getmonero.org` to ``https://www.getmonero.org`, so it's better to always use the latter.

Same for the other occurrencies.

Reply:
Sure

Reply:
:)